checkov/common/util/parser_utils.py

```diff
@@ -356,8 +356,13 @@ def get_current_module_index(full_path: str) -> Optional[int]:
     return tf_index
 
 
-def is_nested(full_path: str) -> bool:
-    return TERRAFORM_NESTED_MODULE_PATH_PREFIX in full_path
+def is_nested(full_path: TFDefinitionKeyType) -> bool:
+    from checkov.terraform.modules.module_objects import TFDefinitionKey
```

Comment:
why local import? import loop in this file?

Reply:
yes